dotcom-rendering/scripts/webpack/gu-stats-report-plugin.js

```diff
@@ -0,0 +1,158 @@
+const fetch = require('node-fetch');
+const os = require('os');
+const { exec } = require('child_process');
+
+class GuStatsReportPlugin {
+	constructor(config) {
+		this.buildName = config?.buildName;
+		this.project = config?.project;
+		this.team = config?.team;
+		this.buildCount = 0;
+		this.gitBranch = null;
```

Comment:
Any reason to be using `null` and `undefined` (from optional chaining above)?

[Interesting article on this](https://writingjavascript.com/why-you-should-always-use-undefined-and-never-null) - besides the argument of which, I find it best to at least stick to one.